### ISSUE TITLE
[Update Request]: Joplin v3.0.13

### ISSUE THREAD

**Issue by shyun3** [microsoft/winget-pkgs#163335]
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

Joplin.Joplin

### Package Version

3.0.13

### Please describe the changes you would like to see

There is a new release (3.0.13) available. Download Page Url: https://github.com/laurent22/joplin/releases

**Comment by aisgbnok** [comment]
Joplin 3.0.13 has been published in WinGet for a week, according to #161967.

Is there something wrong?

```PowerShell
PS E:\> winget search Joplin.Joplin
Name                 Id                        Version Source
--------------------------------------------------------------
Joplin               Joplin.Joplin             3.0.13  winget
Joplin (Pre-release) Joplin.Joplin.Pre-release 2.12.12 winget
PS E:\>
```

**Comment by Trenly** [comment]
[Policy] Needs Author Feedback
